openlibrary/macros/databarWork.html

```diff
@@ -44,6 +52,13 @@ <h3 class="header">
           $if not page.is_fake_record():
               $:render_template('covers/change', page, ".edition-cover .bookCover img")
         </div>
+      <!--This section appears only in mobile/tablet screens-->
+      <div class="mobile-book-header">
+        $if summary!=None:
+           $:summary
+        $:macros.StarRatingsStats(work)
+      </div>
```

Comment:
```suggestion
        $if summary:
          <!--This section appears only in mobile/tablet screens-->
          <div class="mobile-book-header">
             $:summary
          </div>
```
We shouldn't render the `mobile-book-header` div if there is no `summary`.

In Python, a `None` object is considered `False` if it's being evaluated in a conditional statement.  No need to compare the object to `None`!